domains/anmail.is-local.org.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Personal email hosting service",
+
+  "domain": "is-local.org",
+  "subdomain": "anmail",
+
+  "owner": {
+    "email": "wolfo89+anmail@protonmail.com"
+  },
+
+  "record": {
+    "NS": ["ns1.zilore.net", "ns2.zilore.net"]
```

Comment:
Please provide a specific and valid reason for NS records.

Reply:
Dont you allow NS records? In which cases do you allow them

Reply:
We allow them, we just need a valid reasoning behind them where NS actually makes sense. More times than not, users are using NS just to forward their site to a single webpage, in case a normal record like CNAME or A would have been sufficient. If someone has a real reason as to why they need NS records, we allow them.